src/main/java/ru/hse/software/design/CLI.java

```diff
@@ -0,0 +1,72 @@
+package ru.hse.software.design;
+
+import java.util.Scanner;
+
+/**
+ * Class that the user is interacting with.
+ * Internally, it stores the 'isRunning' flag as a private field,
+ * which is set to true when the application is running and false when it terminates.
+ * It has two main public methods 'start' and 'exit'
+ * and private method 'createExecutor' responsible for creating an object of the Executor class.
+ */
+
+public class CLI {
+    private boolean isRunning = false;
+
+    /**
+     * Creates CLI instance and calls its method 'start'.
+     *
+     * @param args input args
+     **/
+    public static void main(String[] args) {
+        System.out.println("-----CLI interpreter-----");
+        CLI instance = new CLI();
```

Comment:
```suggestion
        var instance = new CLI();
```
Хотя бывает мнение, что явный тип улучшает читабельность программы --- но тут он написан дважды, что точно излишество.

Reply:
Fixed)